scripts/spack/packages/spheral/package.py

```diff
@@ -238,7 +248,7 @@ def initconfig_package_entries(self):
 
         entries.append(cmake_cache_path('qhull_DIR', spec['qhull'].prefix))
 
-        entries.append(cmake_cache_path('aneos_DIR', spec['m-aneos'].prefix))
+        entries.append(cmake_cache_path('aneos_DIR', spec['m_aneos'].prefix))
```

Comment:
Package names haven't changed, only the representation on disk has. This should still be with a `-`